modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -1045,6 +1047,10 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
     return this.supportsAddressType(utxolib.bitgo.outputScripts.scriptTypeForChain(chain));
   }
 
+  keyIdsForSigning(): number[] {
```

Comment:
Is this moved here for forward compatibility purposes when non-btc coins might use taproot?

Reply:
No, it is just easier if we always get all three pubkeys in general case. We should not make exceptions for coins just because they don't support taproot ;)

One side benefit is that having the pubkeys at hand allows us to derive all the required scripts before signing, so we do not have to use `redeemScript`/`witnessScript` on the unspent anymore. Plus we can sanity check the pubScript.